### ISSUE TITLE
Broken HTTP header ref examples on redoc-cli version 0.8.5.

### ISSUE THREAD

**Issue by marcoonroad** [Redocly/redoc#998]
The previous pinned version which we have things OK is the redoc-cli `0.7.0`.

The expected / normal behavior on version `0.7.0`:

![Screenshot from 2019-07-31 19-34-32 (copy)](https://user-images.githubusercontent.com/3820407/62253038-07acaa00-b3cb-11e9-8758-e64d9e2f1aa4.png)

The bug (on version `0.8.5`):

![Screenshot from 2019-07-31 19-30-43 (copy)](https://user-images.githubusercontent.com/3820407/62253025-f95e8e00-b3ca-11e9-8999-475ab5287008.png)

The code regarding the HTTP headers:

```yaml
components:
  schemas:
    RawXPartnerIdType:
      description: The API Key identifying our partner.
      type: string
      example: "003026bbc133714df1834b8638bb496e-8f4b3d9a-e931-478d-a994-28a725159ab9"

    RawXAnonymousDeviceIdType:
      description: A unique identification collected from the device running the mobile app.
      type: string
      example: "2fc4b5912826ad1"
``` 

```yaml
parameters:
- name: x-anonymous-device-id
  in: header
  required: true
  schema:
    $ref: 'schemas/primitives.yml#/components/schemas/RawXAnonymousDeviceIdType'
- name: x-partner-id
  in: header
  required: true
  schema:
    $ref: 'schemas/primitives.yml#/components/schemas/RawXPartnerIdType'
```

**Comment by RomanHotsiy** [comment]
fixed in `redoc-cli@0.8.6`

**Comment by marcoonroad** [comment]
Thank you.
